include/usb/usb_ch9.h

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (c) 2020 PHYTEC Messtechnik GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief USB Chapter 9 structures and definitions
+ *
+ * This file contains the USB Chapter 9 structures definitions
+ * and follows, with few exceptions, the USB Specification 2.0.
```

Comment:
You could mention those exceptions here.

Reply:
I will improve the documentation in subsequent PR's.